apps/ts-minbar-test-react-components/src/index.ts

```diff
@@ -37,6 +43,11 @@ async function performTest() {
     fs.mkdirSync(path.join(tempPaths.testApp, 'src'));
     fs.copyFileSync(scaffoldPath('index.tsx'), path.join(tempPaths.testApp, 'src/index.tsx'));
     fs.copyFileSync(scaffoldPath('tsconfig.json'), path.join(tempPaths.testApp, 'tsconfig.json'));
+
+    fs.mkdirSync(path.join(tempPaths.testApp, 'patches/'));
+    fsExtra.copySync(scaffoldPath('patches/'), path.join(tempPaths.testApp, 'patches/'));
```

Comment:
wondering why we need library abstraction for Node native api ? https://nodejs.org/docs/latest-v14.x/api/fs.html#fs_fs_copyfilesync_src_dest_mode

Reply:
because native `fs.copyFile` is not recursive and does not handle directories

Reply:
removed in [b961173](https://github.com/microsoft/fluentui/pull/21828/commits/b9611738e6121f6b609ad8b7cbc34b9d375d4b39)